config.json

```diff
@@ -5,6 +5,7 @@
     "style": {
       "primaryColor": "#24578f"
     },
+    "multilanguage": true,
```

Comment:
In plaats van `multilanguage` zouden we er ook voor kunnen kiezen om hier iets als `supportedLanguages` te zetten en dan de array met instellingen zoals die nu in LanguangeSwitch staat hier te zetten zodat we eventueel meer dan 2 talen/andere talen dan nl/en kunnen ondersteunen